src/app/lite/copy.ml

```diff
@@ -24,28 +24,30 @@ let proof_tooltip_alt =
 
 let state_tooltip =
   [ `Text
-      "The zk-SNARK proof serves as a succinct blockchain,\n           \
-       validating the entire protocol state just as a heavy blockchain does \
-       in existing cryptocurrencies."
+      "The zk-SNARK proof serves as a succinct blockchain,\n\
+      \           validating the entire protocol state just as a heavy \
+       blockchain does in existing cryptocurrencies."
   ; `New_line
   ; `Text "The staged and locked ledger hashes are the "
   ; `Link ("merkle roots", "https://en.wikipedia.org/wiki/Merkle_tree")
   ; `Text
-      " of two versions of the database of accounts.\n           \
-       Changes to accounts are reflected immediately in the staged ledger hash.\n           \
-       The locked ledger hash is set from the staged ledger hash periodically,\n           \
-       as zk-SNARK proofs are computed." ]
+      " of two versions of the database of accounts.\n\
+      \           Changes to accounts are reflected immediately in the staged \
+       ledger hash.\n\
+      \           The locked ledger hash is set from the staged ledger hash \
+       periodically,\n\
+      \           as zk-SNARK proofs are computed." ]
 
 let account_tooltip =
   [ `Text
       "Once a client has a protocol state and a succinct blockchain \
        certifying that state, they can get their account information with a \
-       small amount of additional data.\n             \
-       Namely, they need a merkle-path from the protocol state's ledger hash \
-       to their account."
+       small amount of additional data.\n\
+      \             Namely, they need a merkle-path from the protocol state's \
+       ledger hash to their account."
   ; `New_line
   ; `Text
       "The succinct blockchain, protocol state, merkle-path, and account \
-       information are altogether just a few kilobytes,\n            \
-       so Coda can provide a full proof of the state of an account with just \
-       this tiny amount of data." ]
+       information are altogether just a few kilobytes,\n\
+      \            so Coda can provide a full proof of the state of an \
```

Comment:
can you remove these extra backslashes (I think the ocamlformat barfed here -- the website will look weird if these remain)